presto-main/src/test/java/io/prestosql/sql/planner/TestLiteralEncoder.java

```diff
@@ -106,6 +109,10 @@ public void testEncode()
         assertEncode(utf8Slice("hello"), createVarcharType(13), "CAST('hello' AS varchar(13))");
         assertEncode(utf8Slice("hello"), VARCHAR, "CAST('hello' AS varchar)");
 
+        LongTimestampWithTimeZone timestamp = LongTimestampWithTimeZone.fromEpochSecondsAndFraction(0, 0, TimeZoneKey.UTC_KEY);
+        assertEncode(timestamp, TimestampWithTimeZoneType.createTimestampWithTimeZoneType(6), "TIMESTAMP(6) WITH TIME ZONE '1970-01-01 00:00:00.000000 UTC'");
```

Comment:
- use some other value than 0, like
```
LongTimestampWithTimeZone.fromEpochSecondsAndFraction(123456789, 123456789012L, TimeZoneKey.UTC_KEY)
```

- add a test cases with other precisions as well (eg 0, 3, 6, 9, 12)

- add toString for LongTimestampWithTimeZone, as currently test failure can look like below
  (see `io.prestosql.spi.type.LongTimestamp#toString` for how this particular tostring should look like)

```
java.lang.AssertionError: expected [io.prestosql.spi.type.LongTimestampWithTimeZone@2fc44916] but found [io.prestosql.spi.type.LongTimestampWithTimeZone@2fb83f02]
Expected :io.prestosql.spi.type.LongTimestampWithTimeZone@2fc44916
Actual   :io.prestosql.spi.type.LongTimestampWithTimeZone@2fb83f02
```